west.yml

```diff
@@ -26,7 +26,7 @@ manifest:
   # Please add items below based on alphabetical order
   projects:
     - name: hal_atmel
-      revision: 5690f5b84495c8b657ff204c5c827df1ab9e12e3
+      revision: 72443ed8f1fa39dc3c1c2a5dba0958381106036d
```

Comment:
Need to do `revision: pull/3/head` here because the PR branch is not fetched into the upstream repo.